app/core/store/modules/layoutChrome.js

```diff
@@ -0,0 +1,56 @@
+export default {
+  namespaced: true,
+
+  state () {
+    // TODO: Currently saving volume to session instead of database.
+    let cachedSound
+    if (window.sessionStorage) {
+      cachedSound = window.sessionStorage.getItem('layoutChrome/soundOn')
```

Comment:
This seems like something that is perfect for https://github.com/championswimmer/vuex-persist.  Not necessarily something that we want saved to user but something that we do want to persist in the state.  Maybe better managed by a "user settings" store module that persists itself / maybe clears when the user logs out?